src/Makefile

```diff
@@ -128,7 +128,7 @@ tags:
 	ctags -R
 
 clean:
-	rm -f $(objects) $(deps)
+	rm -f $(objects) $(deps) .version.cc .version.o
```

Comment:
I am not sure we want to remove .version.cc as this is distributed in releases and wont get regenerated out of a git repository, I think we only want to remove .version.o